docker/tarfile/dest.go

```diff
@@ -168,8 +172,12 @@ func (d *Destination) ReapplyBlob(ctx context.Context, info types.BlobInfo) (typ
 }
 
 func (d *Destination) createRepositoriesFile(rootLayerID string) error {
-	repositories := map[string]map[string]string{
-		d.reference.Name(): {d.reference.Tag(): rootLayerID}}
+	repositories := map[string]map[string]string{}
+	if d.repoTags != nil {
+		for _, repoTag := range d.repoTags {
+			repositories[repoTag.Name()] = map[string]string{repoTag.Tag(): rootLayerID}
```

Comment:
I don't think this works for `repoTags` which share a name, e.g. "busybox:tag1", "busybox:tag2"; only one tag would get recorded.

Reply:
Fixed, should record both tags  now.